src/writer.rs

```diff
@@ -230,7 +232,9 @@ pub mod tests {
         }
     }
 
-    fn read_2d_image<P: AsRef<::std::path::Path>>(path: P) -> Array2<f32> {
+    fn read_2d_image<P>(path: P) -> Array2<f32>
+        where P: AsRef<Path>
```

Comment:
Is that how `rustfmt` put it?

Reply:
My bad, sorry. I'm not used to run `rustfmt`! Will do.